packages/runtime/src/helpers/functions.ts

```diff
@@ -41,6 +41,11 @@ export const generateFunctions = async (
   const functionDir = join(functionsDir, HANDLER_FUNCTION_NAME)
   const publishDir = relative(functionDir, publish)
 
+  const nextServerModuleAbsoluteLocation = getNextServerModulePath(appDir)
+  const nextServerModuleRelativeLocation = nextServerModuleAbsoluteLocation
+    ? relative(functionDir, nextServerModuleAbsoluteLocation)
+    : undefined
+
```

Comment:
if `nextServerModuleAbsoluteLocation` is falsy we could warn here that non-SSG routes will fail (tho that might be noisy, so if we want warning like that maybe we add one once we know if we need lambdas for routes)